en/community/user-groups/index.md

```diff
@@ -8,8 +8,7 @@ In the programming community, user groups form support networks for
 people interested in certain topics. They are a great place to increase
 your skills and network with other programmers. User groups are informal
 and their structure varies from group to group. Anyone can form their
-own group and set their own rules and schedule.
-{: .summary}
+own group and set their own rules and schedule{: .summary}
```

Comment:
`{: .summary}` should not be appended to previous line, isn't it?

https://www.ruby-lang.org/en/community/user-groups/
<img width="613" alt="Screenshot 2024-06-19 at 8 26 53" src="https://github.com/ruby/www.ruby-lang.org/assets/7338443/83d21056-af67-48b1-b2e9-c0823ad8c3fd">

🤔

Reply:
Oops! Fixed in https://github.com/ruby/www.ruby-lang.org/pull/3295

Reply:
Thanks~